docs/content/reference/infrastructure/aws.md

```diff
@@ -89,4 +89,7 @@ In this section we want to dissect who creates what and what not. It contains 4
 
 === "Hosted Cluster AWS account"
     - Network Load Balancer for default ingress
-    - S3 bucket for registry
\ No newline at end of file
+    - S3 bucket for registry
+
+!!! note
+For the Private Link networking to work, we've observed that the Endpoint zone in the hosted cluster AWS account, must match the zone of the instance resolved by the Service Endpoint in the management cluster AWS account.
```

Comment:
While this is true, there is no way for the customer to ensure this because AZ designation is unique the AWS account.  us-east-1a in one account might not be the same zone as us-east-1a in another account.